tests/end_to_end/test_ip_reputation/test_ip_reputation.py

```diff
@@ -0,0 +1,76 @@
+import os
+import json
+import re
+import pytest
+from tempfile import gettempdir
+
+from wazuh_testing import end_to_end as e2e
+from wazuh_testing import event_monitor as evm
+from wazuh_testing.tools import configuration as config
+
+
+alerts_json = os.path.join(gettempdir(), 'alerts.json')
+test_data_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'data')
+test_cases_file_path = os.path.join(test_data_path, 'test_cases', 'cases_ip_reputation.yaml')
+configuration_playbooks = ['configuration.yaml']
+events_playbooks = ['generate_events.yaml']
+teardown_playbooks = ['teardown.yaml']
+
+configurations, configuration_metadata, cases_ids = config.get_test_cases_data(test_cases_file_path)
+
+
+@pytest.mark.filterwarnings('ignore::urllib3.exceptions.InsecureRequestWarning')
+@pytest.mark.parametrize('metadata', configuration_metadata, ids=cases_ids)
+def test_ip_reputation(configure_environment, metadata, get_dashboard_credentials, generate_events,
+                       clean_alerts_index):
+    """
+    Test to detect a IP Reputation
+    """
+
```

Comment:
Put a comment to refer to what is the first alert or change the name of first_alert to something that can indicate it.

The same for the case below.

Reply:
Done in ce7b61d